test/appengine/test_gae_manager.py

```diff
@@ -46,7 +47,7 @@ def setup_method(self, method):
     def test_exceptions(self):
         # DeadlineExceededError -> TimeoutError
         with pytest.raises(urllib3.exceptions.TimeoutError):
-            self.pool.request("GET", "/sleep?seconds=0.005", timeout=0.001)
+            self.pool.request("GET", "/sleep?seconds=0.005", timeout=SHORT_TIMEOUT)
```

Comment:
If we implement the above comment, can we have the `seconds=X` call be dynamic? Maybe 2 * SHORT_TIMEOUT?

Reply:
Ah yes, great catch! I think I'll continue to use 5 * SHORT_TIMEOUT